### PR TITLE
add the wunderlist under the Tools for team & Collaboration

### PR DESCRIPTION
**Pull request by FrankBian**
a perfect to-do list app that work seamlessly across all major devices

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,7 @@ Table of Contents
   * https://www.blockspring.com/ - Share scripts with anyone on your team: cross language and with spreadsheet users. Free for 5 million runs a month.
   * https://helpmonks.com/ - Shared inbox for teams - Free for open source projects and non-profit organizations.
   * http://typetalk.in/ - Share and discuss ideas with your team through instant messaging on the web or on your mobile.
+  * https://www.wunderlist.com/ - Share your lists and work collaboratively on projects with your colleagues - Available for free on iPhone,iPad,Mac,Android,Windows and the Web.
```
Comment (ripienaar) [inline README.md]: do you have a link that shows what the terms of the free trial is? stays free with limitations or time based? every link that I tried to get more details seem to insist on a signup

Comment (FrankBian) [inline README.md]: it’s free to download and use , and  also offers a purchase version .  the free version would satisfy the requirement from small team completely. 

https://itunes.apple.com/app/wunderlist-to-do-list-tasks/id410628904 <https://itunes.apple.com/app/wunderlist-to-do-list-tasks/id410628904>
the description in iTunes is below : 
>Wunderlist is free to download and use. Wunderlist Pro upgrades your experience and gives you unlimited access to Files, Assigning and >Subtasks to help you accomplish even more for $4.99 a month or $49.99 a year through an in-app purchase.




Best Regards,
Frank Bian

Comment (ripienaar) [inline README.md]: sweet, thanks

**Comment by ripienaar** [comment]
ah sorry, do you mind rebasing? Had to undo some merges for a service thats broken and that broke yours

**Comment by FrankBian** [comment]
Welcome ,thx


Best Regards,
Frank Bian

**Comment by ripienaar** [comment]
hmm, weird, something still going, will look at merging it manually later

**Comment by FrankBian** [comment]
thanks very much


Best Regards,
Frank Bian

**Comment by FrankBian** [comment]
Hi ,Is the merging  broken by any exceptions ?

What can I do for u?

**Comment by ripienaar** [comment]
ah no github just wont auto merge, i think your rebase was wrong or something hehe, sorry I've been snowed under 

not sure, if you rebased to latest master it should just work

**Comment by bogdal** [comment]
@FrankBian assuming that `origin` is your fork and `upstream` is this repo try this:
```
git fetch upstream
git rebase upstream/master
```

**Comment by ripienaar** [comment]
thanks

**Pull request merged**